README.md

```diff
@@ -208,3 +208,43 @@ class Example extends React.Component {
   }
 }
 ```
+
+- `addVariables` (boolean) - Adding variables for props. Defaults to `false`.
```

Comment:
Instead of `addVariables`, let's call it `exportVariableDeclarations`.

Reply:
Yes, but maybe call `declarePropTypesVariables`? It's not only export:
https://github.com/milesj/babel-plugin-typescript-to-proptypes/pull/1/files#diff-ada6231da2b06b9c22c8023a81419998R15